docs/plugins/gmusic.rst

```diff
@@ -20,34 +20,15 @@ Then, you can enable the ``gmusic`` plugin in your configuration (see
 
 Usage
 -----
-
-To automatically upload all tracks to Google Play Music, add the ``auto: yes``
-parameter to your configuration file like the example below::
-
-    gmusic:
-        auto: yes
-        email: user@example.com
-        password: seekrit
+**Configuration required before use.**
```

Comment:
Hmm—I actually kind of liked having the configuration example up here, since people really will need to do this first. Any particular reason you moved it downward? Can we move it back here to stick with the logical flow of steps?

Reply:
You got it!